action/protocol/staking/protocol.go

```diff
@@ -178,6 +179,27 @@ func (p *Protocol) Start(ctx context.Context) error {
 			return err
 		}
 	}
+
+	// load the candidate center to StateReader
+	srCtx := protocol.MustGetStateReaderCtx(ctx)
+	return srCtx.SR.Load(protocolID, p.inMemCandidates)
```

Comment:
instead the "Load" could be moved into a function like "CreatePreStates", where you can pass sr as extra parameter.

Reply:
Load means write the cand center to SR
in CreatePreStates, we call Unload, which is to read from SR

Reply:
modified such that this is not needed
see new commit "address comment"